### PR TITLE
Update "seek of closed file" fix for current django-storages

### PR DESCRIPTION
**Pull request by nikolas**
Using this class with the new django-storages gives the error:

  ValueError: seek of closed file

This change should fix that -
https://github.com/jschneier/django-storages/issues/382#issuecomment-654984775

### CODE REVIEW

File: cacheds3storage/__init__.py [hunk 1]
```diff
@@ -11,28 +11,31 @@ def __init__(self, *args, **kwargs):
         self.local_storage = get_storage_class(
             'compressor.storage.CompressorFileStorage')()
 
-    # Fix for "ValueError: seek of closed file" problem with boto3
-    # https://github.com/jschneier/django-storages/issues/382#issuecomment-377174808
-    def _save_content(self, obj, content, parameters):
+    # https://github.com/jschneier/django-storages/issues/382
```
Comment (sdreher) [inline cacheds3storage/__init__.py:14]: If this is removed completely, will it just pick up the one in `django-storages`?

Comment (nikolas) [reply cacheds3storage/__init__.py:14]: Hmmmm, good point.. maybe the fix is already in I may be getting confused. Let me see if I can reproduce this locally with `./manage.py compress` to test that.

Comment (nikolas) [reply cacheds3storage/__init__.py:14]: Okay, so when testing this locally (actually with `./manage.py collectstatic`):

When I remove the _save/_save_content override completely, I get the same error:
```
  File "/home/nik/src/d/mediathread/ve/lib64/python3.7/site-packages/django/core/files/base.py", line 55, in chunks
    self.seek(0)
ValueError: seek of closed file
```

With this change.... well it's still running and not showing any output yet. I'll see what happens, but looks promising.

Comment (nikolas) [reply cacheds3storage/__init__.py:14]: Okay, yes. It's actually copying the files successfully with this override (when you do `./manage.py collectstatic -v 3`)... which unfortunately still seems necessary.

**Pull request merged**